app/locker/data/dashboards/LrsDashboard.php

```diff
@@ -52,19 +52,40 @@ public function statementCount(){
    *
    **/
   public function actorCount(){
-
-    $count = $this->db->statements->aggregate(
+    $count_array = ['mbox' => '', 'openid' => '', 'mbox_sha1sum' => '', 'account' => ''];
+    
+    $count_array['mbox'] = $this->db->statements->aggregate(
               array('$match' => $this->getMatch( $this->lrs )),
               array('$group' => array('_id' => '$statement.actor.mbox')),
               array('$group' => array('_id' => 1, 'count' => array('$sum' => 1)))
               );
+    
+    $count_array['openid'] = $this->db->statements->aggregate(
+            array('$match' => $this->getMatch( $this->lrs )),
+            array('$group' => array('_id' => '$statement.actor.openid')),
+            array('$group' => array('_id' => 1, 'count' => array('$sum' => 1)))
+    );
+    
+    $count_array['mbox_sha1sum'] = $this->db->statements->aggregate(
+            array('$match' => $this->getMatch( $this->lrs )),
+            array('$group' => array('_id' => '$statement.actor.mbox_sha1sum')),
+            array('$group' => array('_id' => 1, 'count' => array('$sum' => 1)))
+    );
+    
+    $count_array['account'] = $this->db->statements->aggregate(
+            array('$match' => $this->getMatch( $this->lrs )),
+            array('$group' => array('_id' => '$statement.actor.account.name')),
```

Comment:
This needs changing to include `statement.actor.account.homePage` too. You might be able to do:
```php
array('_id' => array(
  'accountName' => '$statement.actor.account.name',
  'accountHomePage' => '$statement.actor.account.homePage'
)
```